--- Formula/patch-src/awscli.original.rb
+++ Formula/patch-src/awscli.rb
@@ -42,7 +42,9 @@
       }
     EOS
 
-    system libexec/"bin/python3", "scripts/gen-ac-index", "--include-builtin-index"
+    unless ENV["CI"]
+      system libexec/"bin/python3", "scripts/gen-ac-index", "--include-builtin-index"
+    end
   end
 
   def caveats
@@ -54,7 +56,9 @@
 
   test do
     assert_match "topics", shell_output("#{bin}/aws help")
-    assert_include Dir["#{libexec}/lib/python3.8/site-packages/awscli/data/*"],
-                   "#{libexec}/lib/python3.8/site-packages/awscli/data/ac.index"
+    unless ENV["CI"]
+      assert_include Dir["#{libexec}/lib/python3.8/site-packages/awscli/data/*"],
+                     "#{libexec}/lib/python3.8/site-packages/awscli/data/ac.index"
+    end
   end
 end
